model/project_ref.go

```diff
@@ -1078,6 +1078,21 @@ func FindFirstProjectRef() (*ProjectRef, error) {
 	return projectRef, nil
 }
 
+// FindAllMergedTrackedEnabledProjectRefs returns all enabled project refs in the db
+// that are currently being tracked (i.e. their project files
+// still exist and the project is not hidden).
+// Can't hide a repo without hiding the branches, so don't need to aggregate here.
+func FindAllMergedTrackedEnabledProjectRefs() ([]ProjectRef, error) {
+	projectRefs := []ProjectRef{}
+	q := db.Query(bson.M{ProjectRefHiddenKey: bson.M{"$ne": true}, ProjectRefEnabledKey: bson.M{"$ne": false}})
```

Comment:
This will return projects that are enabled _and_ projects that have enabled set to nil (i.e. they default to the repo, or else they're false). It would make sense then to re-iterate through the projects after 1093 to make sure that all the remaining projects _are_ enabled.

Reply:
Also can we split this out into multiple lines to make it cleaner since there are multiple query parameters?